soknad-omsorgspenger-utbetaling/src/main/java/no/nav/k9/søknad/omsorgspenger/utbetaling/OmsorgspengerUtbetalingSøknadValidator.java

```diff
@@ -1,52 +1,44 @@
 package no.nav.k9.søknad.omsorgspenger.utbetaling;
 
+import no.nav.k9.søknad.PeriodeValidator;
 import no.nav.k9.søknad.SøknadValidator;
 import no.nav.k9.søknad.felles.*;
 
-import java.time.ZonedDateTime;
+import javax.validation.Validation;
+import javax.validation.ValidatorFactory;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class OmsorgspengerUtbetalingSøknadValidator extends SøknadValidator<OmsorgspengerUtbetalingSøknad> {
```

Comment:
trenger vi denne som separat klasse?  mesteparten kan håndters med deklarativ javax.validation.
evt. + @AssertTrue for komplisert validering. Det gjør modellen deklarativ og mulig å validere, uten at det ligger validator kode gjemt her?

Reply:
Usikker. Fulgte det som allerede er satt opp her.

Reply:
ok, får ta en annen gang, lite hensiktsmessig med separat validator